MCLauncher/MainWindow.xaml.cs

```diff
@@ -52,6 +53,22 @@ public partial class MainWindow : Window, ICommonVersionCommands {
             });
         }
 
+        private async void Button_Click(object sender, RoutedEventArgs e)
```

Comment:
Please rename this to something that makes more sense, eg. ImportButtonClick

Reply:
Also, code style note:
This project uses inline { and } brackets. Please reconfigure your IDE to follow this codestyle and update the changes.